plugins/action/ci_script.py

```diff
@@ -157,6 +159,11 @@ def run(self, tmp=None, task_vars=None):
         if "cmd" in task_args:
             task_args.pop("cmd")
 
+        # Are we running dry-run?
+        dry_run = False
```

Comment:
i added the dry_run stuff (copied from ci_make.py) mainly to get the tests to pass locally (ci_make ansible tests)

i could remove that and the associated tests and move to a followup. Or if folks don't mind we can keep it here (it is minimal addition unless I missed something further needed here)

Reply:
imho keeping the dry_run capability is a good thing.

Reply:
ok thanks then keeping here